src/utils.ts

```diff
@@ -7,11 +7,27 @@ export const truthy = Boolean as unknown as <T>(
 
 function parseRepository(repository: string) {
```

Comment:
Did a little searching, seems like switching to use [hosted-git-info](https://www.npmjs.com/package/hosted-git-info) would be better than reimplementing this ourselves.

Reply:
Thanks for taking care of this, @mskelton. I apologize for not following up myself.

Reply:
No worries at all!